modules/data.atmosphere/R/tdm_predict_subdaily_met.R

```diff
@@ -67,6 +67,9 @@ predict_subdaily_met <- function(outfolder, in.path, in.prefix, path.train, dire
   
   n.ens <- length(ens.labs)
   
+  # Update in.path with our prefix (seems silly, but helps with parallelization)
+  in.path <- file.path(in.path, in.prefix)
+  
   # Extract the lat/lon info from the first of the source files
   fnow <- dir(in.path, ".nc")[1]
   ncT <- ncdf4::nc_open(file.path(in.path, fnow))
```

Comment:
Looking at the line added above, I think this will now do something a bit different from what PEcAn has expected up to now, as it will make the file name ```in.path/in.prefix/fnow``` while we've more typically thought of prefix as the start of the filename (e.g. ```in.path/in.prefix-year.nc```) . I'm not sure that this is necessarily 'wrong' because we also haven't been dealing with ensembles yet, and I can't recall off the top of my head whether we'd settled on ensemble number being a subfolder or part of the filename. I'm guessing this code is doing the latter.